beacon-chain/core/blocks/block_operations_test.go

```diff
@@ -899,6 +875,16 @@ func TestProcessBlockAttestations_BlockRootFailure(t *testing.T) {
 			},
 		},
 	}
+	justifiedBlock := &pb.BeaconBlock{
+		Slot: helpers.StartSlot(params.BeaconConfig().GenesisEpoch + 1),
```

Comment:
```suggestion
		Slot: helpers.StartSlot(params.BeaconConfig().GenesisEpoch + 1) - 2, // Imagine 2 skip blocks
```

This should cause the test to fail if anyone were to remove the iterative lookup